shop/src/main/java/ro/msg/learning/shop/Controller/ProductController.java

```diff
@@ -0,0 +1,47 @@
+package ro.msg.learning.shop.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.DTO.ProductAndProductCategoryDto;
+import ro.msg.learning.shop.DTO.ProductCategorySimpleDto;
+import ro.msg.learning.shop.Service.Implementation.ProductCategoryService;
+import ro.msg.learning.shop.Service.Implementation.ProductService;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequestMapping("/product")
+@RestController
+@Validated
```

Comment:
What is this annotation doing? Please check a little bit more if we need it at the moment.